Dockerfile

```diff
@@ -1,11 +1,13 @@
 FROM python:3.9.2-slim-buster as base
+RUN pip install --upgrade pip
 RUN pip install poetry
 COPY poetry.lock pyproject.toml ./
 
 FROM base as production
 RUN poetry config virtualenvs.create false --local && poetry install
 RUN poetry add gunicorn
-RUN poetry install
+RUN poetry add pymongo
+RUN poetry add pymongo[srv]
```

Comment:
You don't need these lines - you've already got pymongo[srv] listed (check your `pyproject.toml`) so it will be installed by the `poetry install` above; currently your docker output says here:
> The following packages are already present in the pyproject.toml and will be skipped:
  • pymongo